Sources/AccountSDKIOSWeb/Simplified Login/SimplifiedLoginViewController.swift

```diff
@@ -1,15 +1,48 @@
 import SwiftUI
 import UIKit
 
-public struct SimplifiedLoginViewModel {
+public struct SimplifiedLoginUIFactory {
+    
+    public static func buildViewController() -> UIViewController {
+        let clientRedirectURI = URL(string: "com.sdk-example.pre.602504e1b41fa31789a95aa7:/login")!
+        let clientConfiguration = ClientConfiguration(environment: .pre,
```

Comment:
We also dont want hardcoded values for client configuration. So another TODO ;)

Reply:
Removed it